templates/email.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title></title>
+</head>
+<body>
+{{ surname }}
+{{ name }}
+
+ <a href= 'http://localhost:8000/registration/verification/{{ user.id }}/'><span>here</span></a>
```

Comment:
Can you replace `localhost` with {{ request.get_host }} ?
http://stackoverflow.com/questions/1451138/how-can-i-get-the-domain-name-of-my-site-within-a-django-template